### PR TITLE
Add checkbox function

### PR DESCRIPTION
**Pull request by keroloussamy**
In this PR, I made the following
1- Using modules to separate the code.
2- Add checkbox function.
3- Using localStorage to save data.

### CODE REVIEW

File: src/completedTasks.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { getList, storeList } from './localStorage.js';
+
+export default function completed(e) {
+  const list = getList();
```
Comment (Dipeshtwis) [inline src/completedTasks.js]: - [x] I would recommend using the proper ES6 syntax to fulfill the project requirement and also writing clean code as ES6 syntax covers more work in less code.

File: src/localStorage.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+export function setSortingIndex(todoList) {
+  return todoList.map((item, index) => { item.index = index; return item; });
+}
+
+export function storeList(todoList) {
+  localStorage.setItem('todo-list', JSON.stringify(todoList));
+}
+
+export function getList() {
+  return JSON.parse(localStorage.getItem('todo-list'));
```
Comment (Dipeshtwis) [inline src/localStorage.js]: - [x] Here as well. I would recommend using the proper ES6 syntax to fulfill the project requirement and also writing clean code as ES6 syntax covers more work in less code.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @keroloussamy, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/localStorage.js (hunk 2 above)
Comment (keroloussamy) [reply src/localStorage.js]: Done

File: src/completedTasks.js (hunk 1 above)
Comment (keroloussamy) [reply src/completedTasks.js]: Done

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @keroloussamy,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**